### ISSUE TITLE
Update Changelog with progress since 0.3.12

### ISSUE THREAD

**Issue by jrr** [airbrake/airbrake-js#193]
The latest airbrake-js release (as of this writing) is v0.5.8 from December 2015. The Changelog was last updated for v0.3.12 in March 2015.

I don't need detail for every single release, but @vmihailenco could you summarize what changed over this time? Here's a GitHub compare view:

https://github.com/airbrake/airbrake-js/compare/v0.3.12...v0.5.8

**Comment by vmihailenco** [comment]
From what I see from https://github.com/airbrake/airbrake-js/compare/v0.3.12...v0.5.8 there are mostly bug fixes and API changes in order to make notifier work with requirejs/browserify/wiredep. I don't see anything important or very interesting.

**Comment by jrr** [comment]
Having just updated the version in our app, I can point out _several_ important changes that were relevant for our project:
- Package name changed from `airbrake-js` to `airbrake-js-client`
- The source file to import from bower_modules changed from `airbrake.js` to e.g. `dist/client.min.js`
- The interface to using the library changed significantly:
 - Initialization: change from e.g. `Airbrake.setProject()` to `new airbrakeJs.Client()`
 - Sending an error: change from `Airbrake.push(err)` to e.g. `airbrake.notify(err)`

These are the kinds of things that users of the library really need to know as they maintain their production apps.

Here's a good reference about the what and the why: http://keepachangelog.com/

**Comment by zefer** [comment]
@jrr we'll get it fixed as soon as we can, thanks for bringing it to our attention. We'd also welcome a PR.

**Comment by shime** [comment]
@jrr package name has changed a lot earlier (in c88597a33a5)
